data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/processor/LambdaProcessorConfig.java

```diff
@@ -46,15 +40,15 @@ public class LambdaProcessorConfig {
 
     @JsonPropertyDescription("invocation type defines the way we want to call lambda function")
     @JsonProperty("invocation_type")
-    private String invocationType = REQUEST_RESPONSE;
+    private String invocationType;
 
-    @JsonPropertyDescription("sdk timeout defines the time sdk maintains the connection to the client before timing out")
-    @JsonProperty("sdk_timeout")
-    private Duration sdkTimeout = DEFAULT_SDK_TIMEOUT;
+    @JsonPropertyDescription("Defines the way Data Prepper treats the response from Lambda")
+    @JsonProperty("response_cardinality")
+    private String responseCardinality;
```

Comment:
```suggestion
    private ResponseCardinality responseCardinality;
```

You will need to make other changes.